packages/rancher-istio-1.8/charts/app-readme.md

```diff
@@ -35,11 +35,19 @@ http://{{ .Values.nameOverride }}-grafana.{{ .Values.namespaceOverride }}.svc:{{
 ```
 **Custom Tracing Installation with Kiali**
 
-To use a custom Tracing installation, set the `kiali.external_services.tracing` url and update the `.Values.tracing.contextPath` in the rancher-istio values.yaml.
+The `kiali.external_services.tracing.url`, `kiali.external_services.tracing.in_cluster_url` and `.Values.tracing.contextPath` is set in the rancher-istio values.yaml.
 
-This url depends on the values for `namespaceOverride`, and `.Values.service.externalPort` in your rancher-tracing or other tracing instance.:
+To ensure traces work for your services in the Kiali dashboard, you must set `kiali.external_services.tracing.url` and `kiali.external_services.tracing.in_cluster_url` key value using the complete external url to your tracing instance.
 ```
-http://tracing.{{ .Values.namespaceOverride }}.svc:{{ .Values.service.externalPort }}/{{ .Values.tracing.contextPath }}
+https://<your domain>/k8s/clusters/<cluster id>/api/v1/namespaces/{{ .Values.namespaceOverride }}/services/http:{{ .Values.namespaceOverride }}.svc:{{ .Values.service.externalPort }}/{{ .Values.tracing.contextPath }}
 ```
+An example with all the values filled in:
+```
+https://111.222.333.44/k8s/clusters/c-zbpdp/api/v1/namespaces/istio-system/services/http:tracing:16686/proxy/jaeger
+```
+The url depends on the default values for `namespaceOverride`, and `.Values.service.externalPort` being set in your rancher-tracing or other tracing instance.
+
+> **Note:** If you are having a hard time determining the complete url value, deploy Istio with Kiali and Tracing enabled.
+Navigate to the Jaeger dashboard and copy the url from the browser up to the {{ .Values.tracing.contextPath }} context path to use as your complete url value. The context path is `/jaeger` by default.
```

Comment:
This is a lot to put into the app-readme. Ideally that page is quite short. I would put the example and the note in the readme, not in the app-readme.

Reply:
I have removed the example and notes section from the app-readme. It is now just in the readme.